README.md

```diff
@@ -134,7 +134,10 @@ Read more about [Netlify redirects](https://docs.netlify.com/routing/redirects/)
 
 ### Versions
 
-You can check our `package.json` for supported Next and Node versions. Our support of Next 10 is currently experimental.
+You can check our `package.json` for supported Next and Node versions.
+
+For breaking Next.js releases, all Next.js versions before the breaking release will be locked to the last working plugin version.
+You can check dependency version compatibility [here](https://github.com/netlify/plugins/blob/master/site/plugins.json).
```

Comment:
Similar comment - can we make it easier for our users so they don't have to search for the `compatibility` field in our `plugins.json` (how would they know to search for that specific field?)